documentation/UserGuide.md

```diff
@@ -10,6 +10,10 @@ User Guide - squad-leader-android
 
 Squad Leader runs on Android devices version 4.0.1 and higher. Viewshed analysis works on Android 4.1 and higher.
 
+#### Uncheck "Don't Keep Activities"
+
+On the Android device, go to Settings > Developer Options or Settings > General > Developer Options. (Some devices don't have Developer Options available by default. If your device does not have Developer Options, you can skip this step.) If your device has Developer Options available, ensure that **"Don't keep activites"** or **"Do not keep activities"** is **unchecked**. If that option is checked, the map will reset whenever the user leaves it, such as when the spot report form or the settings dialog appears.
```

Comment:
When this setting is checked, Android destroys the Activity when hidden so that it has to be recreated when displayed again. Yes, any Activity containing an ArcGIS MapView would be affected. This setting is intended only for development and is not the default, so I don't know why Jim's device had it checked.